platform/janitor/src/org/netbeans/modules/janitor/Janitor.java

```diff
@@ -58,11 +58,34 @@
     "# {1} - the days of abandonement",
     "# {2} - the disk space can be reclaimed (in megabytes)",
     "TIT_ABANDONED_USERDIR=NetBeans {0} was last used {1} days ago.",
+
+    "# {0} - the name of the abandoned cache dir.",
+    "# {1} - the disk space can be reclaimed (in megabytes)",
+    "TIT_ABANDONED_CACHEDIR=NetBeans cache directory {0} seems to be abandoned.",
+
     "# {0} - is the user directory name",
     "# {1} - the days of abandonement",
     "# {2} - the disk space can be reclaimed (in megabytes)",
     "DESC_ABANDONED_USERDIR=Remove unused data and cache directories of NetBeans {0}. "
             + "Free up {2} MB of disk space.",
+
+    "# {0} - is the cache directory name",
+    "# {1} - the disk space can be reclaimed (in megabytes)",
+    "DESC_ABANDONED_CACHEDIR=NetBeans does not found a user dir for "
+            + "cache dir {0}, so it maybe abandoned. "
```

Comment:
Agree with @mbien - something along lines of "NetBeans could not find a user dir for cache dir {0}, so it is probably abandoned".

At least "maybe" should be "may be" or "might be" - "NetBeans could not find a user dir for cache dir {0}, so it may be abandoned".

Reply:
Thank you for the feedback! Changed the message, I hope that reads better, and feel free to correct me!

Reply:
wait, you made it worse :D Just take what Neil wrote and paste it into the file. He lives directly at the source (I think), we can trust him.

Reply:
It seems I'm suffering from some tunnel vision. Never noticed the first line of Neil's comment.

Fixed now. Thank you @mbien !